src/notification/telegram.ts

```diff
@@ -15,13 +15,18 @@ export function sendTelegramMessage(link: Link, store: Store) {
 
 		(async () => {
 			const givenUrl = link.cartUrl ? link.cartUrl : link.url;
+			const results = [];
 
-			try {
-				await client.sendMessage(telegram.chatId, `${Print.inStock(link, store)}\n${givenUrl}`);
-				logger.info('✔ telegram message sent');
-			} catch (error) {
-				logger.error('✖ couldn\'t send telegram message', error);
+			for (const chatId of telegram.chatId.split(',')) {
```

Comment:
I'd add a note in the telegram doc in `README`. I'd also update `config.ts` to use `envOrArray()`.